README.md

```diff
@@ -3,3 +3,19 @@
 This repository is cataloging University Library access to scholarly literature.
 Scholarly articles are identified using their DOIs.
 The impetus for this project was [this discussion](https://github.com/greenelab/scihub-manuscript/issues/21 "Potential followup: comparison to authorized access") on the Sci-Hub Coverage Study.
+
+# License
+
+The data files in this repository are released under the CC0 1.0 public domain dedication ([`LICENSE-CC0.md`](LICENSE-CC0.md)). These include files matched by the following glob patterns:
+
+- `*.tsv`
+- `*.sqlite`
+- `*.xml`
```

Comment:
These globs won't match compressed versions, which is something we'll likely have.

Reply:
Do you mean, e.g., `.gz`? If so, do you have a lab standard for compression?

Reply:
> If so, do you have a lab standard for compression?

I like `.xz` since it achieves the best compression ratio (although it does take a long time to produce). No standard. We can make this choice later.